src/components/markdown/LanguageSection.tsx

```diff
@@ -5,16 +5,68 @@ import { LANGUAGE_LABELS } from '../../context/UserDataContext/properties/userLa
 
 export const LanguageSection = props => {
   const { lang: userLang } = useContext(UserDataContext);
+  let expand = null;
+  if (typeof props.expandable == 'undefined') {
+    expand = true;
+  } else {
+    expand = false; // try this
+  }
 
   const sections = {};
   React.Children.map(props.children, child => {
+    const oldChild = child;
+    let newChild = React.cloneElement(child, {
+      children: oldChild.props.children,
+      mdxType: oldChild.props.mdxType,
+      originalType: oldChild.props.originalType,
+      expandable: expand,
+    });
     const type = child.props.mdxType;
     const typeToLang = {
       CPPSection: 'cpp',
       JavaSection: 'java',
       PySection: 'py',
     };
-    sections[typeToLang[type]] = child;
+
+    newChild.props.children = React.Children.map(
+      newChild.props.children,
+      child => {
+        let ogChild = child;
+        if (typeof ogChild.props != 'undefined') {
+          const { oldProps } = ogChild.props;
+          let newC = React.cloneElement(child, {
+            ...oldProps,
+            expandable: expand,
+          });
+
+          newC.props.children = React.Children.map(
+            newC.props.children,
+            child2 => {
+              const ogChild2 = child2;
+              if (typeof ogChild2.props != 'undefined') {
+                const { oldProps2 } = ogChild2.props;
+                return React.cloneElement(child2, {
+                  ...oldProps2,
+                  expandable: expand,
+                });
+              } else {
+                return child2;
+              }
+            }
+          );
+
+          return newC;
```

Comment:
Try not to reuse variable names already declared in a higher scope. This can easily cause confusion.
Also, `newC` is not very descriptive, so I recommend changing it to something like `newGrandchild`.

```suggestion
      grandchild => {
        let oldGrandchild = grandchild;
        if (typeof oldGrandchild.props != 'undefined') {
          const { oldProps } = oldGrandchild.props;
          let newGrandchild = React.cloneElement(grandchild, {
            ...oldProps,
            expandable: expand,
          });

          newGrandchild.props.children = React.Children.map(
            newGrandchild.props.children,
            child2 => {
              const ogChild2 = child2;
              if (typeof ogChild2.props != 'undefined') {
                const { oldProps2 } = ogChild2.props;
                return React.cloneElement(child2, {
                  ...oldProps2,
                  expandable: expand,
                });
              } else {
                return child2;
              }
            }
          );

          return newGrandchild;
```